floodwatch/src/js/components/Compare.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {FilterParent} from './FilterParent';
+import AdBreakdown from '../../stubbed_data/adbreakdown_response.json';
+import Filters from '../../stubbed_data/filter_response.json';
+
+// import '../../Compare.css';
+
+type CompareState = {}
+export class Compare extends Component {
+  state: CompareState;
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Row>
+        <Col xs={12}>
+        <CompareContainer/>
+        </Col>
+      </Row>
+    )
+  }
+}
+
+type CompareContainerState = {
+  leftOptions: JSON;
```

Comment:
`JSON` is actually `Object`. You should define a type `Options` somewhere that contains what the options actually are. That type will then later be useful when building the options object via a query builder and passing it to the API.